### PR TITLE
Localization

### PR DESCRIPTION
**Pull request by samisuteria**
- Actual localization

</img>
</img>

### CODE REVIEW

File: Steps4ImpactTests/LocalizationSpec.swift [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+**/
+
+import Quick
+import Nimble
+@testable import Steps4Impact
+
+class LocalizationSpec: QuickSpec {
+  override func spec() {
+    describe("Localization") {
+
```
Comment (compnerd) [inline Steps4ImpactTests/LocalizationSpec.swift]: unnecessary whitespace

**Review by compnerd — APPROVED** [review]

**Comment by samisuteria** [comment]
Passed one of the checks - hopefully azure pipelines fixes itself ¯\_(ツ)_/¯
